client/types/KeyguardRequests.d.ts

```diff
@@ -0,0 +1,125 @@
+declare namespace KeyguardRequests {
```

Comment:
What do you think about calling it `KeyguardRequest` (singular), just like `KeyguardCommand`? In the code it would then be more correct when referring to a single request type.

Reply:
sounds reasonable :)